src/Files.App/Actions/Show/ToggleSidebarAction.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal sealed class ToggleSidebarAction : ObservableObject, IToggleAction
+	{
+		private IAppearanceSettingsService AppearanceSettingsService { get; } = Ioc.Default.GetRequiredService<IAppearanceSettingsService>();
```

Comment:
```suggestion
		private readonly IAppearanceSettingsService AppearanceSettingsService = Ioc.Default.GetRequiredService<IAppearanceSettingsService>();
```

Reply:
Are you sure? That seems to be in conflict with how all the other similar actions are defined currently.

Reply:
Oh, no, my mistake. Most don't do what I did. It's only the `ToggleToolbarAction` I used as reference that does this.

Reply:
Yeah for private variables, we use fields.